tools/amd_build/build_amd.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+import sys
+import argparse
+sys.path.append(os.path.realpath(os.path.join(
+    __file__,
+    os.path.pardir,
+    os.path.pardir,
+    os.path.pardir,
+    'third_party')))
+
+from hipify import hipify_python
+
+parser = argparse.ArgumentParser(description='Top-level script for HIPifying, filling in most common parameters')
+parser.add_argument(
+    '--project-directory',
+    type=str,
+    default=os.path.normpath(os.path.join(
+        os.path.realpath(__file__),
+        os.pardir,
+        os.pardir,
+        os.pardir,
+    )),
+    help="The root of the project. (default: %(default)s)",
+    required=False)
+
+parser.add_argument(
+    '--output-directory',
+    type=str,
+    default=os.path.normpath(os.path.join(
```

Comment:
As for this, I'd suggest getting rid of the contrived default value here. It could take a default value of `args.project_directory` after the args have been parsed.

Reply:
done